docs/source/integrations/ios-fairplay-sdk.md

```diff
@@ -1,138 +1,140 @@
+2. Navigate to your target and add vudrmFairPlay.framework to your ‘Embedded Binaries’. You should see that vudrmFairPlay.framework also now appears under ‘Linked Frameworks and Libraries’:
+
+### INFORMATION ABOUT APPLICATION TRANSPORT SECURITY (ATS)
+
+iOS/tvOS 9 introduces Application Transport Security (ATS) which restricts the use of insecure HTTP.
+In order to permit playback of content over insecure HTTP exemptions need to be added into your application's Info.plist. For example to disable ATS for any connection you would add the following into your application's Info.plist:
+<dict>
+<key>NSAppTransportSecurity</key> <dict>
 <key>NSAllowsArbitraryLoads</key> <true/>
-</dict> 
-</dict> 
-More information about ATS can be found in Apple’s TechNote https://developer.apple.com/library/prerelease/ios/technotes/App-Transport-Security-Technote/ 
+</dict>
+</dict>
+More information about ATS can be found in Apple’s TechNote https://developer.apple.com/library/prerelease/ios/technotes/App-Transport-Security-Technote/
 
-### EXAMPLE FRAMEWORK USAGE ### 
+### EXAMPLE FRAMEWORK USAGE
 
-1. Import the VUDRMFairPlay module into your player import vudrmFairPlay 
+1. Import the VUDRMFairPlay module into your player import vudrmFairPlay
 
 2. Initialise an instance of VUDRMFairPlay:
 
-Online Playback / Download and Offline Playback: 
-Pass in a reference to your AVPlayer instance, the token to be used, and the asset ID when retrieving a license. 
+Online Playback / Download and Offline Playback:
+Pass in a reference to your AVPlayer instance, the token to be used, and the asset ID when retrieving a license.
 The example uses a single dynamically assigned instance. It would be expected that your application will manage the applications offline assets, including their ID’s, locations and download status. This method uses AVAssetDownloadURLSession with a URLSessionConfiguration to create an AVAssetDownloadTask for each configured asset.
- 
+
 The current example initialises an AVAssetDownloadURLSession with a URLSessionConfiguration upon
 loading the ViewController:
- 
+
 backgroundConfiguration = URLSessionConfiguration.background(withIdentifier: "AssetDownloadConfigurationIdentifier")
-        	assetUrlSession = AVAssetDownloadURLSession(configuration: backgroundConfiguration!,
+assetUrlSession = AVAssetDownloadURLSession(configuration: backgroundConfiguration!,
 assetDownloadDelegate: self, delegateQueue: .main)
- 
+
 The asset configuration is then assigned upon use of the download or
 play buttons. To play an asset we create the assetDownloadTask and then assign the associated urlAsset to a player:
- 
+
 let asset = AVURLAsset(url: assetURL!)
 self.drm = vudrmFairPlay(asset: asset, assetName: assetName, token: token)
-               assetDownloadTask = assetUrlSession.makeAssetDownloadTask(asset: asset, assetTitle: assetName,
+assetDownloadTask = assetUrlSession.makeAssetDownloadTask(asset: asset, assetTitle: assetName,
```

Comment:
Surround code snippet in triple back-ticks

Reply:
any changes to this document will have to be done by Adam